geth/jail/README.md

```diff
@@ -29,73 +29,83 @@ Otto virtual machine and lives forever, but that may change in the future.
 ++-------++ ++-------++ ++-------++  ++-------++
 ```
 
-##Cells
+## Cells
 Each Cell object embeds *VM from 'jail/vm' for concurrency safe wrapper around
 *otto.VM functions. This is important when dealing with setTimeout and Fetch API
 functions (see below).
 
-##Get and Set
+## Get and Set
 (*VM).Get/Set functions provide transparent and concurrently safe wrappers for
 Otto VM Get and Set functions respectively. See Otto documentation for usage examples:
 https://godoc.org/github.com/robertkrimen/otto
 
-##Call and Run
+## Call and Run
 (*VM).Call/Run functions allows executing arbitrary JS in the cell. They're also
-wrappers arount Otto VM functions of the same name. Run accepts raw JS strings for execution,
-Call takes a JS function name (defined in VM) and parameters.
+wrappers around Otto VM functions of the same name. `Run` accepts raw JS strings for execution while `Call` takes a JS function name (defined in VM) and parameters.
 
-##Timeouts and intervals support
-Default Otto VM interpreter doesn't support setTimeout()/setInterval() JS functions,
+## Timeouts and intervals support
+Default Otto VM interpreter doesn't support setTimeout() / setInterval() JS functions,
 because they're not part of ECMA-262 spec, but properties of the window object in browser.
-We add support for them using own implementation of Event Loop, heavily based on http://github.com/status-im/ottoext package. See loop/fetch/promise packages under `jail/internal/`.
 
-Each cell starts a new loop in a separate goroutine, registers functions for setTimeout/setInterval
-calls and associate them with this loop. All JS code executed as callback to setTimeout/setInterval
-will be handled by this loop.
+We add support for them using own implementation of Event Loop, heavily based on [ottoext package](https://github.com/deoxxa/ottoext). See loop/fetch/promise packages under [jail/internal/](https://github.com/status-im/status-go/tree/develop/geth/jail/internal).
+
+Each cell starts a new loop in a separate goroutine, registers functions for setTimeout / setInterval calls and associate them with this loop. All JS code executed as callback to setTimeout/setInterval will be handled by this loop.
 
 For example, following code:
 
-```
+```go
 cell.Run(`setTimeout(function(){ value = "42" }, 2000);`)
 ```
 
-will execute setTimeout and return immidiately, but callback function will
+will execute setTimeout and return immediately, but callback function will
 be executed after 2 seconds in the loop that was started upon current cell.
 
 In order to capture response one may use following approach:
 
-```
+```go
 err = cell.Set("__captureResponse", func(val string) otto.Value {
-	fmt.Println("Captured response from callback:", val)
-	return otto.UndefinedValue()
+
+```
```

Comment:
Somehow this has rendered improperly. Have you used `autoreadme` tool to convert from doc.go to README?

Reply:
Hey @divan good catch. Not sure what happened here maybe I ran auroreadme without the `-template` flag.

May be a good idea to have a Makefile or even a git hook so any change on the go.doc file automatically generates the README.

Anyway that's fixed now

Reply:
I'm afraid it's the problem with upstream `autoreadme`. I used this patched one: https://github.com/divan/autoreadme. Could you please try it?

Reply:
With your fork:

Running `autoreadme -f -template=doc.go` produces no changes.

On the other hand simply running `autoreadme -f` produces an strange result with problems on some code blocks:

<img width="672" alt="screen shot 2018-02-07 at 14 07 58" src="https://user-images.githubusercontent.com/593270/35917961-58c6a936-0c10-11e8-9d15-3a8e2941a757.png">